src/Amalgam/Amalgam.h

```diff
@@ -15,8 +16,17 @@
 
 extern "C"
 {
+	//status from LoadEntity
+	typedef struct
+	{
+		bool loaded;
+		const char *message;
+		const char *version;
+	} LoadEntityStatus;
+	#define LOADENTITYSTATUS_DEFAULT { true, "", "" }
```

Comment:
So at the C API level it can't be. It has to be pure C to go across the library boundary safely. But it is clunky to have this used in the C++ interface so I'll create a C++ one that is much easier to work with and handles resources correctly.

FWIW: this struct worked well in my first implementation but not so much after I refactored.

Reply:
Created a C++ status for cleaner usage internally with a converter at the C/C++ boundary.